modules/lib/frontend-component-factory.js

```diff
@@ -56,11 +57,12 @@ function FrontendComponentFactory(db, deps) {
 
 var proto = FrontendComponentFactory.prototype;
 
-proto.createComponent = function(type) {
-
+proto.createComponent = function (type) {
+    console.debug('FrontentComponentFactory::createComponent', type);
     var component = this._createComponent(type);
+
     if (!component) {
-        return;
+        return false;
```

Comment:
Why false? `return;` returns `undefined` which is cleaner because we really have nothing to return. `false` should only be returned when the other case would be `true` as a return value.